CodeListLibrary_project/clinicalcode/views/View.py

```diff
@@ -102,7 +102,7 @@ def about_pages(request, pg_name=None):
             return render(request, 'clinicalcode/brand/HDRUK/about/team.html', {})
 
         elif pg_name.lower() == "hdruk_about_technical_details".lower():
-            return render(request, 'clinicalcode/brand/HDRUK/about/technical-details.html', {})
+            return technicalpage(request)
```

Comment:
@ZinnurovArtur 
where is technicalpage(request)?

Reply:
On the same view file